pkg/types/timestamp/v0.0.1/timestamp_v0_0_1_schema.json

```diff
@@ -0,0 +1,48 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "http://rekor.sigstore.dev/types/timestamp/timestamp_v0_0_1_schema.json",
+    "title": "Timestamp v0.0.1 Schema",
+    "description": "Schema for RFC3161 entries",
+    "type": "object",
+    "properties": {
+        "tsr": {
+            "description": "Information about the tsr file associated with the entry",
+            "type": "object",
+            "properties" : {
+                "content": {
+                    "description": "Specifies the tsr file content inline within the document",
+                    "type": "string",
+                    "format": "byte"
+                },
+                "hash": {
```

Comment:
it's not clear to me that storing the hash of the tsr has any value? We can just generate it and use it for indexing? But I dunno, so it's in here for now.

Reply:
Yeah I think indexing is the main use case.